src/client/components/AdminPanel/index.js

```diff
@@ -13,8 +13,6 @@ const AdminPanel = () => {
     <Tabs
       variant="secondary"
       onRequestTabChange={(event, { index }) => setSelectedIndex(index)}
-      minHeight="10rem"
-      maxHeight="10rem"
     >
       <Tabs.Panel
         renderTitle="Course Reserve Listings"

```

Comment:
@kilometerskang Should be renamed to "Course resource listing" to match how the other tabs are capitalized.